guides/v2.3/graphql/develop/resolvers.md

```diff
@@ -7,8 +7,8 @@ github_link: graphql/develop/resolvers.md
 
 A resolver performs GraphQL request processing. In general, it is responsible for constructing a query, fetching data and performing any calculations, then transforming the fetched and calculated data into a GraphQL array format. Finally, it returns the results wrapped by a callable function.
 
-A GraphQL resolver must implement the `\Magento\Framework\GraphQl\Query\ResolverInterface` interface. This interface contains the requirement to return a `\Magento\Framework\GraphQl\Query\Resolver\Value`. This value takes in a callable function to its constructor that will be invoked at the latest possible time for the resolver to require its data. As a result, a list of items being resolved can be retrieved all at once by establishing a buffer that contains all relevant parent data to filter and fetch for the children list data.
+A GraphQL resolver must implement the [`\Magento\Framework\GraphQl\Query\ResolverInterface`](https://github.com/magento/magento2/tree/2.3-develop/lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php) interface. This interface contains the requirement to return a [`\Magento\Framework\GraphQl\Query\Resolver\Value`](https://github.com/magento/magento2/tree/2.3-develop/lib/internal/Magento/Framework/GraphQl/Query/Resolver/Value.php). This value takes in a callable function to its constructor that will be invoked at the latest possible time for the resolver to require its data. As a result, a list of items being resolved can be retrieved all at once by establishing a buffer that contains all relevant parent data to filter and fetch for the children list data.
```

Comment:
Change all instances of https://github.com/magento/magento2/tree/2.3-develop

to 

https://github.com/magento/magento2/blob/{{ page.guide_version }}-develop

Reply:
Ok thanks, I will do it. Thanks for your support.